echo-telemetry/src/main/java/com/netflix/spinnaker/echo/telemetry/TelemetryEventListener.java

```diff
@@ -202,6 +193,14 @@ private String hash(String clearText) {
     return Hashing.sha256().hashString(clearText, StandardCharsets.UTF_8).toString();
   }
 
+  private static EnumValueDescriptor parseEnum(EnumDescriptor ed, String value) {
```

Comment:
The fact that we have to call this a few times (and also get a generic `EnumValueDescriptor` that loses type information) makes me wonder if there's a more standard way of doing this with protos?  Maybe there's a way to parse and default without needing to manually do this?  @plumpy may have thoughts here.

Reply:
I don't think the loss of type information matters as it's passed to the enum's `valueOf()` method that accepts a generic `EnumValueDescriptor`.

Unfortunately there aren't many examples or SO posts out there of how to better parse proto Enums, and I've never worked with Java protos internally.